src/Array/push.js

```diff
@@ -0,0 +1,14 @@
+
+Array.prototype.myPush = function(elementToAdd) {
+    this[this.length] = elementToAdd;    
```

Comment:
Does this automatically increases `this.length` by one?

Reply:
The functionality is correct. 
Yes, it does increase the length by one.

Reply:
have added tests

Reply:
please see the new commit below